src/components/Login/Panels/PhonePanel.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import PhoneForm from '@/components/Login/PhoneForm/'
+
+const PhonePanel = () => {
+  /* TODO: add 'auth update' functionality */
```

Comment:
What's this referring to? Something for down the line or no longer required?

Reply:
This was from the previous pr, not sure what it means but I've removed it.